src/test/url-loader/url-resolver_test.ts

```diff
@@ -20,9 +20,8 @@ import {UrlResolver} from '../../url-loader/url-resolver';
 class SimplestUrlResolver extends UrlResolver {
   resolve(
       firstUrl: ResolvedUrl|PackageRelativeUrl, secondUrl?: FileRelativeUrl) {
-    const [baseUrl, url] = secondUrl === undefined ?
-        [this.brandAsResolved('/test/'), firstUrl as PackageRelativeUrl] :
-        [this.brandAsResolved(firstUrl), secondUrl];
+    const [baseUrl = '/test/' as ResolvedUrl, url] =
```

Comment:
Woah, I had no idea that syntax was legal.

I'm a little weirded out by it, but it does make sense. Would it be clearer if `getBaseAndUnresolved`  took a third argument (maybe optionally?), a resolved package url?

Reply:
Personally, I like this better because the code itself demonstrates expectation of a possible undefined and it can be handled how the client wants?  I'll merge as is but if it still feels weird we can make the change.